.gitignore

```diff
@@ -48,3 +48,6 @@ typings/
 
 # dotenv environment variables file
 .env
+
+# Editor configurations
+.vscode
```

Comment:
We probably want to add *.log at some point

Reply:
Good point. I'll add when it comes up